common.mk

```diff
@@ -45,7 +51,8 @@ $(FIRRTL_FILE) $(ANNO_FILE): generator_temp
 	@echo "" > /dev/null
 
 # AG: must re-elaborate if ariane sources have changed... otherwise just run firrtl compile
-generator_temp: $(SCALA_SOURCES) $(ARIANE_VLOG_SOURCES) $(sim_files)
+generator_temp: $(SCALA_SOURCES) $(PROJECT_GENERATOR_SOURCES) $(sim_files)
+	echo $(PROJECT_GENERATOR_SOURCES)
```

Comment:
Ignore these `echo`s. They are just for debugging purposes. I will remove them when this is ready to merge.